ui/src/main.tsx

```diff
@@ -13,9 +13,34 @@ const App = React.lazy(() =>
   })),
 );
 
+/**
+ * Since the z-index of other plug-ins is set to 1111111,
+ * part of the content of our Drawer component is covered,
+ * so the z-index is adjusted uniformly and multiplied by 10000 based on the default value.
+ */
+const zIndices = {
+  hide: -1,
+  auto: "auto",
+  base: 0,
+  docked: 10,
+  dropdown: 10000000,
+  sticky: 11000000,
+  banner: 12000000,
+  overlay: 13000000,
+  modal: 14000000,
+  popover: 15000000,
+  skipLink: 16000000,
+  toast: 17000000,
+  tooltip: 18000000,
+};
```

Comment:
let's not change so many of it, let's keep the scope of this change small. only the files drawer, it's a bit risky to overwrite so many styles

Reply:
https://chakra-ui.com/docs/styled-system/theme#z-index-values